CHANGELOG.md

```diff
@@ -10,6 +10,75 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 -->
 <!--Start Of Releases (DO NOT DELETE THIS LINE)-->
 
+## [2.0.0a4] - 2023-01-26
+
+## Highlights
+
+This is the fourth alpha release for 2.0.0. It contains a fix for the renamed dbt exceptions in dbt version 1.4.0.
```

Comment:
We should also mention that a major performance issue with the 2.0 dbt templater was fixed. At least one user was having big issues due to this.

(You can see PR #4273 for more details, but I wouldn't bother getting that specific in the highlights.)

Reply:
I would also mention @WittierDinosaur's work to improve parse performance of large SQL files (#4278)